.readthedocs.yaml

```diff
@@ -4,10 +4,12 @@ build:
   os: "ubuntu-22.04"
   tools:
     python: "3.12"
-
-python:
-  install:
-    - requirements: docs/requirements.txt
+  jobs:
+    post_create_environment:
+      - pip install poetry
+      - poetry config virtualenvs.create false
+    post_install:
+      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH poetry install --with dev
```

Comment:
Dumb question -- where is this environment variable set? I must be missing it in the GitHub repo settings.

Reply:
When ReadTheDocs kicks off the documentation build, it first creates a virtual environment and installs stuff, then it sets the `READTHEDOCS_VIRTUALENV_PATH`. See: https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_VIRTUALENV_PATH